soc/silabs/common/soc_power_pmgr.c

```diff
@@ -35,10 +34,8 @@ void pm_state_set(enum pm_state state, uint8_t substate_id)
 		energy_mode = SL_POWER_MANAGER_EM1;
 		break;
 	case PM_STATE_SUSPEND_TO_IDLE:
-		energy_mode = SL_POWER_MANAGER_EM2;
-		break;
 	case PM_STATE_STANDBY:
-		energy_mode = SL_POWER_MANAGER_EM3;
+		energy_mode = SL_POWER_MANAGER_EM2;
```

Comment:
Do you believe we should raise a warning if a device still use the oscillator while the user request PM_STATE_STANDBY (BTW, are we technically able to raise a such warning)?

Reply:
The user doesn't request a PM state, the system selects the lowest allowed state. The standby case could technically be removed from here since it can't happen when there is no `power-states` node for it in devicetree (even `pm_state_force()` requires the state to be declared in devicetree).